<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngagementType</class>
 <widget class="QWidget" name="EngagementType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Report Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="mButtonGAH">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Geometric Action History</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="mButtonTD">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Threat Deployment</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="mButtonDFH">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Damage Factor History</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="mButtonThSm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Threat Summary</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="mButtonMFP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Missile Flight Profile</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="mButtonIS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Interval Statistics</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="mButtonPAPA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>PAPA</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="mButtonEFL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Engagement Failure Log</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QRadioButton" name="mButtonTStats">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Track Statistics</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QRadioButton" name="mButtonFPWH">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Flight Profile Waypoint History</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QRadioButton" name="mButtonWE">
        <property name="text">
         <string>Weapon Expenditures</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="mButtonTErH">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Track Error History</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="mButtonTMS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Threat Mode Summary</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="mButtonTEvH">
        <property name="text">
         <string>Track Event History</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="mButtonET">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Event Timing</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QRadioButton" name="mButtonTS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Track States</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="mButtonActing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Summary by Acting</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QRadioButton" name="mButtonAction">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Summary by Action</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Report Options</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
